scripts/add-cahps-measures.js

```diff
@@ -0,0 +1,91 @@
+// Libraries
+const fs    = require('fs');
+const parse = require('csv-parse');
+const path  = require('path');
+
+const additionalMeasuresFilepath = '../util/additional-measures.json'
+var additionalMeasures = require(additionalMeasuresFilepath);
+
+// We want to overwrite CAHPS measures every time we run this script.
+additionalMeasures = additionalMeasures.filter(function(measure) {
+  var re = /CAHPS_\d+/i;
+  return measure.measureId.match(re) === null;
+});
+
+// Constants
+const CAHPS_CSV_COLUMNS = [
+  'Measure Name',
+  'Measure Points',
+  'Measure Contribution'
```

Comment:
I tried and failed. But I added a comment to make it clear that they are not being used and why I left them in.